src/main/java/uk/gov/pay/api/model/CardDetails.java

```diff
@@ -30,7 +30,7 @@
 
     @JsonProperty("card_brand")
     private final String cardBrand;
-    
+
```

Comment:
@kbottla There are a few places in my PR where the spacing/tabbing in actual code has changed, probably because of a setting in my text editor. Will these need to be fixed?